.gitignore

```diff
@@ -27,3 +27,4 @@ build-stamp
 .pytest_cache/
 .mypy_cache/
 .benchmarks/
+venv/
```

Comment:
You should add that in your global gitignore not in each project's gitignore https://stackoverflow.com/a/7335487/2519059